test/libyul/KnowledgeBaseTest.cpp

```diff
@@ -58,7 +58,7 @@ class KnowledgeBaseTest
 		for (auto const& [name, expression]: m_ssaValues.values())
 			m_values[name].value = expression;
 
-		return KnowledgeBase(m_dialect, m_values);
+		return KnowledgeBase(m_dialect, [this] -> decltype(auto) { return m_values; });
```

Comment:
```suggestion
		return KnowledgeBase(m_dialect, [this]() -> decltype(auto) { return m_values; });
```
also here should finally do it, I'd hope.